service/flags.go

```diff
@@ -17,25 +17,41 @@ package service // import "go.opentelemetry.io/collector/service"
 import (
 	"errors"
 	"flag"
-	"strings"
-
+	"fmt"
+	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/featuregate"
+	"gopkg.in/yaml.v3"
+	"strings"
 )
 
 const (
-	configFlag = "config"
+	configFlag    = "config"
+	buildInfoFlag = "build-info"
 )
 
 var (
 	// Command-line flag that control the configuration file.
 	gatesList = featuregate.FlagValue{}
+	BuildFlag bool
 )
 
 type configFlagValue struct {
 	values []string
 	sets   []string
 }
 
+type componentsOutput struct {
+	Version string
+
```

Comment:
Can you remove the extraneous empty lines?

Reply:
Ok, changing that